extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -12,24 +13,35 @@ import {
   StateProperty,
   getStoreService,
 } from '../../../amp-story/1.0/amp-story-store-service';
+import {AmpStoryShoppingAttachment} from '../amp-story-shopping-attachment';
+import '../amp-story-shopping-tag';
+import '../../../amp-story-page-attachment/0.1/amp-story-page-attachment';
 
 describes.realWin(
   'amp-story-shopping-tag-v0.1',
   {
     amp: {
       runtimeOn: true,
-      extensions: ['amp-story-shopping:0.1'],
+      extensions: ['amp-story-shopping:0.1', 'amp-story-page-attachment:0.1'],
     },
   },
   (env) => {
     let win;
-    let element;
+    let shoppingTagEl;
     let shoppingTag;
     let storeService;
     let localizationService;
 
     beforeEach(async () => {
       win = env.win;
+
+      // Set up the story.
+      const storyEl = win.document.createElement('amp-story');
+      const pageEl = win.document.createElement('amp-story-page');
+      storyEl.getAmpDoc = () => new AmpDocSingle(win);
```

Comment:
Nit: I don't know if this can be an issue, but instead of creating 2 ampdocs here and in line 68 you could do `const ampdoc = new AmpDocSingle(win)` and use it in both places.